packages/apputils-extension/src/index.tsx

```diff
@@ -154,11 +166,54 @@ const translatorConnector: JupyterFrontEndPlugin<ITranslatorConnector> = {
   },
 };
 
-const plugins: JupyterFrontEndPlugin<any>[] = [
+/**
+ * The workspace manager plugin.
+ */
+const workspaceManagerPlugin: ServiceManagerPlugin<Workspace.IManager> = {
+  id: '@jupyterlite/apputils-extension:workspace-manager',
+  description: 'The workspace manager plugin.',
+  autoStart: true,
+  provides: IWorkspaceManager,
+  requires: [ILocalForage],
+  optional: [],
+  activate: (_: null, forage: ILocalForage): Workspace.IManager => {
+    const defaultStorageName = 'JupyterLite Storage - Workspaces';
+    const storageName =
+      PageConfig.getOption('contentsStorageName') || defaultStorageName;
```

Comment:
Consider:
- naming this something new (e.g. `workspaceStorageName`) to separate it from `settings` and `contents`
- adding to the schema